lib/android/audio.nit

```diff
@@ -231,8 +252,12 @@ end
 class MediaPlayer
 	private var nmedia_player: NativeMediaPlayer is noinit
 
+	# Used to control the state of the mediaplayer,
```

Comment:
As explained in the commentary of the class, the mediaplayer has a state diagram and the user should look at the android doc about it to respect some specifications.